QUICKSTART.md

```diff
@@ -0,0 +1,306 @@
+# OpenTelemetry QuickStart
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Configuration](#configuration)
+    + [Sampler](#sampler)
+    + [Span Processor](#span-processor)
+    + [Exporter](#exporter)
+- [Tracing](#tracing)
+  * [Create basic Span](#create-basic-span)
+  * [Create nested Spans](#create-nested-spans)
+  * [Span Attributes](#span-attributes)
+  * [Create Spans with events](#create-spans-with-events)
+  * [Create Spans with links](#create-spans-with-links)
+  * [Context Propagation](#context-propagation)
+- [Metrics](#metrics)
+
+<!-- tocstop -->
+
+OpenTelemetry can be used to instrument code for collecting distributed traces and recording metrics.
+For more details, check out the [Specification Overview].
+
+In the following examples, we demonstrate how to configure the OpenTelemetry SDK, create spans and record metrics through the OpenTelemetry API.
+
+[Specification Overview]: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/overview.md
+
+# Configuration
+
+**Libraries** that want to export distributed tracing using OpenTelemetry must only take dependency on the `opentelemetry-api` package
```

Comment:
```suggestion
**Libraries** that want to export distributed tracing using OpenTelemetry only need a dependency on the `opentelemetry-api` package
```

Reply:
I think the intent was to make clear its discouraged (or even prohibited by the spec) to access the SDK as an instrumenting library as only the application should set that up.